CHANGELOG.md

```diff
@@ -2,7 +2,30 @@
 
 ## [Unreleased]
 
-Complete deletion `protoc-gen-buf-check-breaking` and `protoc-gen-buf-check-lint`, which have been moved to `protoc-gen-buf-breaking` and `protoc-gen-buf-lint`.
```

Comment:
I'm gonna want to clean this up even further but that's fine, just as long as everything is noted down.